docs/spec_draft.md

```diff
@@ -336,6 +337,47 @@ For boolean tensors, computes the logical operation.
 
 [Back to Ops](#index-of-ops)
 
+## stablehlo.atan2
+
+### Semantics
+
+Performs element-wise atan2 operation on `lhs` and `rhs` tensor and produces a
+`result` tensor, implementing the `atan2` operation from the IEEE-754
+specification. For complex element types, it computes a complex atan2 function
+defined below, with corner cases TBD. Numeric precision is
+implementation-defined. `log` and `sqrt` operations correspond to
+[stablehlo.log](#stablehlolog) and [stablehlo.sqrt](#stablehlosqrt):
+$$atan2(lhs, rhs) = -i * \log\left(\frac{rhs + i * lhs}{sqrt(rhs^{2} + lhs^{2})}
+\right)$$
+
+### Inputs
+
+| Name  | Type                                     |
+|-------|------------------------------------------|
+| `lhs` | tensor of floating-point or complex type |
+| `rhs` | tensor of floating-point or complex type |
+
+### Outputs
+
+| Name     | Type                                     |
+|----------|------------------------------------------|
+| `result` | tensor of floating-point or complex type |
```

Comment:
[shape_inference.cc](https://github.com/tensorflow/tensorflow/blob/2c1b3efe573befdda3f3db399eebb875e690082c/tensorflow/compiler/xla/service/shape_inference.cc#L1028-L1043) does allow integers, although [elemental_ir_emitter.cc](https://github.com/tensorflow/tensorflow/blob/2c1b3efe573befdda3f3db399eebb875e690082c/tensorflow/compiler/xla/service/elemental_ir_emitter.cc#L1868-L1870) does not. Which piece of infrastructure is right about this?

Reply:
Most math operations do not allow integers, so `shape_inference.cc` is likely in the wrong here. `Tanh`, which is the most similar op to this one, though a unary op, does not allow integers. Should I file an issue and set Type Inference to `revisit`?

Reply:
Sounds good to keep the status quo, i.e. set Verification (not Type Inference) to "revisit" and keep #499 open. No further comments on this PR - LGTM.